urllib3/response.py

```diff
@@ -126,8 +129,8 @@ def __init__(self, body='', headers=None, status=0, version=0, reason=None,
         # Are we using the chunked-style of transfer encoding?
         self.chunked = False
         self.chunk_left = None
-        tr_enc = self.headers.get('transfer-encoding', '')
-        if tr_enc.lower() == "chunked":
+        tr_enc = self.headers.get('transfer-encoding', '').lower()
+        if "chunked" in set(tr_enc.split(", ")):
```

Comment:
Lookups, yes. But set instantiation is way slower (lower bound is the same amount of work as just checking membership in a list). In this case, we'd be instantiating every time we're looking up, so it's not buying us anything. :)